test/bin/scenario.sh

```diff
@@ -1189,6 +1205,12 @@ action_run() {
 
     check_dependencies
 
+    if [ $# -eq 0 ]; then
+        RUN_HOST_OVERRIDE=""
```

Comment:
```suggestion

    if [ $# -eq 0 ]; then
        RUN_HOST_OVERRIDE=""
        check_dependencies
```

Move check_suggestion here to avoid starting the ngingx server when we only want to run RF tests